spec/nanoc/cli_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+describe Nanoc::CLI do
+  let(:all_commands) do
+    ObjectSpace.each_object(Cri::Command)
```

Comment:
Yeah…

For `Nanoc::CLI`, I could’ve probably changed that to `Nanoc::CLI.root_command` and then write a function to (recursively) find subcommands, but that’s arguably not much better…